scripts/generic-ci-tests.sh

```diff
@@ -168,7 +168,18 @@ case "$TEST_SUITE" in
         cp -R $HOME/firefox/ firefox/
         export SELENIUM_FIREFOX_PATH=firefox/firefox
 
-        PAVER_ARGS="-n $NUMBER_OF_BOKCHOY_THREADS --with-flaky --with-xunit"
+        case "$NUMBER_OF_BOKCHOY_THREADS" in
```

Comment:
Could you add a comment about this, seeing that it could be set as an environment variable. Aside from that :+1: